src/Files.App/Actions/Content/Archives/CompressIntoSevenZipAction.cs

```diff
@@ -37,7 +37,7 @@ public async Task ExecuteAsync()
 				FileFormat = ArchiveFormats.SevenZip,
 			};
 
-			await ArchiveHelpers.CompressArchiveAsync(creator);
```

Comment:
I'm a bit worry about this kind of changes (removing `async` from a method that actually has `await` statements), because it will omit some frames from the stack trace and lead to poor diagnostics.

Reply:
@0x5bfa @d2dyno1

Reply:
> omit some frames from the stack trace and lead to poor diagnostics.

I’m not sure how that’ll affect. The await should be attached when that async method have one or more try statement(s). Unless it, I believe we are able to remove await.

Reply:
@hez2010 how would you go about this?

Reply:
👍

Reply:
I don't think changing that will affect much in terms of diagnosability, any crash or issue would most likely be in `ArchiveHelpers.CompressArchiveAsync`.